test/Compiler.test.js

```diff
@@ -261,6 +261,25 @@ describe("Compiler", () => {
 			done();
 		});
 	});
+	it("should not emit on errors (watch)", function(done) {
+		const compiler = webpack({
+			context: __dirname,
+			mode: "production",
+			entry: "./missing",
+			output: {
+				path: "/",
```

Comment:
Maybe another `path` should be used. If webpack (wrongly) outputs here, the write will fail because of permissions and the test will pass.

Reply:
It write to a memory filesystem.

Reply:
My bad